### ISSUE TITLE
Heading and speed returns -1 in IOS

### ISSUE THREAD

**Issue by bonymbose32** [Agontuk/react-native-geolocation-service#366]

**Comment by bonymbose32** [comment]
Can anyone help

**Comment by andrew-nguyen-mcs** [comment]
Having this issue as well.  Anyone know what's happening here?  It doesn't happen on all iOS devices, just only on some, and we can't decipher any rhyme or reason why it would work on some but not others.  We're thinking it's a settings issue, but all of the settings are the same between our iOS devices.

**Comment by matheusyuri7** [comment]
I'm facing the same issue

**Comment by matheusyuri7** [comment]
@bonymbose32 @andrew-nguyen-mcs have you guys found a solution for this issue?

**Comment by MRadek55** [comment]
Did u test it on real scenario? Should work

**Comment by andrew-nguyen-mcs** [comment]
> 

Yeah, for us, we figured out iPads don't have gps functionality if they don't have wifi & cellular built into them.  So that's why we got -1 for the speed and heading.

Here's a thread on it.
https://discussions.apple.com/thread/252416967
